.github/dependabot.yml

```
@@ -7,3 +7,9 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types: ["version-update:semver-patch"]
+  
+  - package-ecosystem: "npm"
+    directory: "/eng/common/"
```

```suggestion
    directory: "/eng/common/tsp-client/"
```

Do you want to update everything under eng/common, or just tsp-client?

---

Yes, Wes suggested to do that. https://github.com/Azure/azure-sdk-tools/pull/12060#discussion_r2355925829

---

Will update it if it won't work.